lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -3174,19 +3130,8 @@ ParseResult FIRStmtParser::parseNode() {
                        moduleContext.targetsInModule, initializerType);
 
   auto sym = getSymbolIfRequired(annotations, id);
-  auto isNamed =
-      !getConstants().options.disableNamePreservation && ::isNamed(id);
-  auto result = builder.create<NodeOp>(initializer.getType(), initializer,
-                                       isNamed ? (Twine("_") + id).str() : id,
-                                       annotations, sym);
-  // If the node is named, then add a debug tap.
-  //
-  // TODO: Change this once the FIRRTL spec supports "named" vs. "unnamed"
-  // nodes.
-  if (isNamed)
-    builder.create<NodeOp>(
-        initializer.getType(), result, id, getConstants().emptyArrayAttr,
-        StringAttr::get(annotations.getContext(), modNameSpace.newName(id)));
+  //sym = getSymbolIfMissing(sym, id);
```

Comment:
Nit: probably can drop this commented-out line, same for below instance of same.